src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -217,6 +220,29 @@ var TestRenderer = ReactFiberReconciler({
     setTimeout(fn, 0, {timeRemaining: Infinity});
   },
 
+  mockComponent(component: Fiber, rootContainer: Container) {
```

Comment:
We should not be exposing the Fiber data structure to renderers. I have some refactoring ideas where this will break. Instead, pass the individual pieces as arguments.